src/DynamoCore/Engine/EngineController.cs

```diff
@@ -38,6 +38,11 @@ public class EngineController : LogSourceBase, IAstNodeContainer, IDisposable
         /// </summary>
         public event AstBuiltEventHandler AstBuilt;
 
+        /// <summary>
+        /// The event notifies client that the VMLibraries have been reset and the VM is now ready to run the new code. 
+        /// </summary>
+        public static event Action VMLibrariesReset;
```

Comment:
what do you think about making this internal? Is it useful to anyone but us?

Reply:
do we want to continue supporting it?

Reply:
I made this internal for now as it is being using only inside the DynamoCore package. Even after we refactor that workflow, we still might need this event.